package.json

```diff
@@ -136,16 +135,16 @@
     "hastscript": "^7.0.2",
     "hoist-non-react-statics": "^3.0.1",
     "io-ts": "^2.2.16",
-    "jail-monkey": "^2.3.2",
+    "jail-monkey": "^2.8.0",
     "jwk-thumbprint": "^0.1.4",
     "lodash": "^4.17.21",
     "metro-babel-register": "^0.72.1",
     "mixpanel-react-native": "2.4.1",
     "pako": "^2.1.0",
     "path-browserify": "0.0.0",
     "pdf-lib": "^1.17.1",
-    "react": "18.1.0",
-    "react-native": "0.70.15",
+    "react": "18.2.0",
+    "react-native": "0.72.14",
```

Comment:
There's already a new version, shall we upgrade to 0.72.15?

Reply:
there's a bug on pod scripts that actually blocks the pod install process, I had to revert the commit where I updated unluckily

ref. https://github.com/facebook/react-native/issues/45036